### PR TITLE
Rubies: 2.7.7, 3.0.5, 3.1.3, 3.2.0-preview2 and 3.2.0-preview3

### PR DESCRIPTION
**Pull request by hurricup**
Updated list of supported rubies

### CODE REVIEW

**Review by skaes — DISMISSED** [review]
LGTM, but I'm not really a maintainer, maybe someone else should review this too?

**Review by havenwood — DISMISSED** [review]
Thank you for the PR!

**Review by havenwood — APPROVED** [review]

**Pull request merged**
